tools/swift-api-digester/swift-api-digester.cpp

```diff
@@ -1003,6 +1003,13 @@ class PrunePass : public MatchedNodeListener, public SDKTreeDiffPass {
           if (D->hasFixedBinaryOrder()) {
             D->emitDiag(diag::decl_added);
           }
+          // Diagnose the missing of @available attributes.
+          // Decls with @_alwaysEmitIntoClient aren't required to have an
+          // @available attribute.
+          if (D->getIntroducingVersion().empty() &&
+              !D->hasDeclAttribute(DeclAttrKind::DAK_AlwaysEmitIntoClient)) {
+            D->emitDiag(diag::new_decl_without_intro);
```

Comment:
Can this be an optional diag? It's only relevant for APIs shipped in Apple OSs.

Reply:
Is this really necessary? Most frameworks using this tool are in such case.